### PR TITLE
In 139834803 cleanup swagger and api

### PR DESCRIPTION
**Pull request by NateOSI**

### CODE REVIEW

File: spec/repositories/person_repository_spec.rb [hunk 1]
```diff
@@ -4,25 +4,21 @@
...
-        .and_yield(mock_request)
-        .and_return(mock_response)
-      expect(mock_request).to receive(:url).with(PersonRepository::PEOPLE_PATH)
-      expect(mock_request).to receive(:headers).and_return({})
-      expect(mock_request).to receive(:body=).with(new_person.to_json)
-      expect(Person).to receive(:new).with(mock_response.body)
-        .and_return(created_person)
-      expect(described_class.create(new_person)).to eq(created_person)
+      new_person = FactoryGirl.build(:person, first_name: 'Homer')
+      new_person_attributes = new_person.to_json
```
Comment (pemaOSI) [inline spec/repositories/person_repository_spec.rb]: new_person has id associated with it. It is create so it should not submit id.

File: spec/features/person/address/edit_address_spec.rb [hunk 2]
```diff
@@ -70,7 +70,7 @@
     person.addresses.first.type = 'Home'
 
     stub_request(:put, api_person_path(person.id))
-      .with(body: person.to_json)
+      .with(body: person.to_json(except: :id))
```
Comment (pemaOSI) [inline spec/features/person/address/edit_address_spec.rb]: put request should have id? is that not the case?

**Comment by pemaOSI** [comment]
looks good to me

File: app/repositories/participant_repository.rb [hunk 3]
```diff
@@ -7,7 +7,7 @@ class ParticipantRepository
   CONTENT_TYPE = 'application/json'
 
   def self.create(participant)
-    response = make_api_call(PARTICIPANTS_PATH, :post, participant)
+    response = make_api_call(PARTICIPANTS_PATH, :post, participant.as_json(except: :id))
```
Comment (ThomasOSI) [inline app/repositories/participant_repository.rb]: Please consider using JSON Serializers to exclude the `:id` attribute. It would be more readable.

**Pull request merged**